code/feature_extraction/ImageHelper.py

```diff
@@ -3,26 +3,34 @@
 except ImportError:
     import Image
 import pytesseract
-import os
+from pathlib import Path
 
 # Make sure that Tesseract is defined even if it is not in $PATH variable
 pytesseract.pytesseract.tesseract_cmd = r'C:\Program Files\Tesseract-OCR\tesseract'
 
 
 class ImageHelper:
-    @staticmethod
-    def get_text(image_path):
-        # Check if path is not a list from pandas
-        if not isinstance(image_path, str):
-            image_path = image_path[0]
+    def __init__(self, data_path):
+        project_path = Path(__file__).parent.parent

```

Comment:
Same, seems a little dirty, we could just pass this as an input argument to the class if needed.